medicines/import/README.md

```diff
@@ -51,6 +52,14 @@ Navigate to the correct directory and run the following command:
 $  cargo run par -d /path/to/pdfs/
 ```
 
-### Verbosity options
+### Other Command Line Options
+
+#### Verbosity (-v, -vv)
 
 Passing `-v` or `-vv` will show more information about the running process at the cost of a slight hit to performance.
+
+**Note:** The progress bar will not be shown if verbosity is enabled.
+
+#### Dry Runs (-r)
```

Comment:
I would think `-n` is more natural as a dry-run flag, but it's no biggy.

Reply:
Agreed, will change this.